packages/client/package.json

```diff
@@ -95,13 +95,11 @@
     "styled-components": "5.3.9"
   },
   "overrides": {
-    "eslint": "9.9.0",
```

Comment:
need for eslint override gone with ESLint 9-compatible import plugin

Reply:
next still required due to OF WS libs incorrect categorisation of deps / numerous vulnerabilities due to the old version of next that they specify 🤦🏻